examples/mosaic-integration/src/VizzesPane.tsx

```diff
@@ -1,12 +1,15 @@
 import { useCallback, useEffect, useState } from 'react';
 import { Viz } from './Viz';
 import './VizzesPane.css';
-import { MarkTypesViz, NYPDComplaintsViz } from './vizzes';
+import { MarkTypesViz, NYPDComplaintsViz, GaiaStarCatalogViz, EarthquakeViz, SeattleWeatherViz } from './vizzes';
 import { FlightsViz } from './vizzes/FlightsViz';
 
 const vizMap: Record<string, Viz> = {
   'Mark Types': new MarkTypesViz(),
   'NYPD Complaints': new NYPDComplaintsViz(),
+  'Gaia Star Catalog': new GaiaStarCatalogViz(),
```

Comment:
Since this is slow to load, we might want to place it lower in this list (after Flights 10M), just so folks have a good experience with the first few before trying this one.